docs/usage.md

```diff
@@ -78,6 +84,7 @@ nextflow run nf-core/nanoseq \
     --flowcell FLO-MIN106 \
     --kit SQK-LSK109 \
     --barcode_kit SQK-PBK004 \
+    --aligner minimap2 \
```

Comment:
This will be the default anyway so best to just leave it out so we can have a minimal usage command.